pkg/apiclient/client_http.go

```diff
@@ -56,7 +58,16 @@ func (c *ApiClient) Do(ctx context.Context, req *http.Request, v interface{}) (*
 	if c.UserAgent != "" {
 		req.Header.Add("User-Agent", c.UserAgent)
 	}
+	if log.GetLevel() >= log.TraceLevel {
+		dump, _ := httputil.DumpRequest(req, true)
+		log.Tracef("request: %s", string(dump))
+	}
+
 	resp, err := c.client.Do(req)
+	if log.GetLevel() >= log.TraceLevel {
```

Comment:
is this dump redundant with the same code in `refreshJwtToken()` used by JWTtransport ? if yes, maybe we can remove it from the `refreshJwtToken()` ?